molecule/kafka-connect-replicator-plain-kerberos-rhel/converge.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Remove the Zookeeper, Kafka Broker, Schema Registry and Kafka Rest Groups from Inventory
-  hosts: zookeeper:kafka_broker:schema_registry:kafka_rest
+  hosts: zookeeper:kafka_controller:kafka_broker:schema_registry:kafka_rest
```

Comment:
Similar to above. I haven't gone through the logic here. But should we have kafka_connect, kafka_controller at both places?